doc/source/contributing.rst

```diff
@@ -173,23 +173,21 @@ For a python 3 environment::
 This will create the new environment, and not touch any of your existing environments,
```

Comment:
On the line above (can't comment there) there is a python 3.4. I think we can simply remove the "3.4" and conda should give you the latest version (or otherwise change with 3.7)